Automattic/vip-go-mu-plugins
# Search: Fix commands get-indexes and get-mapping

## Description
This PR does two things:
1) Removes system operation indexes from being returned in `wp vip-search get-indexes`
2) For multisites, it fixes an error that is returned in `wp vip-search get-mapping` when one indexable is enabled on a subsite, but not another

The error returned from 2) is because features on VIP are activated by a site-basis, rather than network-wide. So, the assumption that each site will have the same indexables does not apply at https://github.com/10up/ElasticPress/blob/043ef0dc6ef1865644da5a33bd68d5838f7ee4e3/includes/classes/Command.php#L443-L465

## Changelog Description

### Plugin Updated: Enterprise Search

Search: Fix commands get-indexes and get-mapping

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
For 1):
1) Do `wp vip-search get-indexes` and see `.geoip_databases` returned from it
2) Apply PR
3) Repeat step 1) and see no `.geoip_databases`

For 2):
1) Create a multisite with two subsites
2) For subsite 1 enable the `terms` feature via `wp vip-search activate-feature terms`
3) Do a network-wide re-index `wp vip-search index --setup --network-wide`
4) `wp vip-search get-mapping` and see an error returned, as it is attempting to query for the terms index on subsite 2
5) Apply PR and repeat step 4) and expect to see the mapping settings returned properly
